inference-engine/src/mkldnn_plugin/nodes/mkldnn_reorder_node.cpp

```diff
@@ -65,41 +65,69 @@ void MKLDNNReorderNode::initSupportedPrimitiveDescriptors() {
     }
 
     supportedPrimitiveDescriptors.emplace_back(config, impl_desc_type::reorder);
-}
 
-void MKLDNNReorderNode::createPrimitive() {
-    auto &dstMemPtr = getChildEdgeAt(0)->getMemoryPtr();
-    auto &srcMemPtr = getParentEdgeAt(0)->getMemoryPtr();
-    if (!dstMemPtr || !dstMemPtr->GetPrimitivePtr())
-        IE_THROW() << "Destination memory didn't allocate.";
-    if (!srcMemPtr || !srcMemPtr->GetPrimitivePtr())
-        IE_THROW() << "Input memory didn't allocate.";
-    if (getSelectedPrimitiveDescriptor() == nullptr)
-        IE_THROW() << "Preferable primitive descriptor is not set.";
-
-    auto inDims = srcMemPtr->GetShape().getStaticDims();
+    isDynamic = !(config.inConfs[0].desc->isDefined() && config.outConfs[0].desc->isDefined());
```

Comment:
I would say that this parameter should be initialized in constructor.

Reply:
as I can see it's not possible for reorder node, because when we create reorder node we don't initialize input/output shapes

Reply:
add comments